Conversion.java

```diff
@@ -0,0 +1,83 @@
+/***
+ * This class has been created to convert the given number
+ * from its base to any other base.
+ * Owner Name :- KRITIKA KUMARI
+ * Date of Creation:- 25 September 2024
+ */
+package NumberConversionAndOperations;
+
+public class Conversion {
+
+    // To convert the given number in decimal form
+    // Return Type -> String
+    // Parameters -> String, Integer
+    final public String otherToDecimal(String number, int base) {
+        long sum = 0;

```

Comment:
sum of what ??? use the rsult value name for which you are using the sum variable